software/tests/test_graphs.py

```diff
@@ -102,15 +104,13 @@ def test_needlessDomainIncludes(self):
     ndi1_results = self.rdflib_data.query(ndi1)
     if (len(ndi1_results) > 0):
         for row in ndi1_results:
-            warn = "WARNING property %s defining domain, %s, [which is subclassOf] %s unnecessarily" % (row["prop"],row["c1"],row["c2"])
-            #warnings.append(warn)
-            log.info(warn + "\n")
+            warn = "property %s defining domain, %s, [which is subclassOf] %s unnecessarily" % (row["prop"],row["c1"],row["c2"])
```

Comment:
could be folded into the log statementr itself.

Reply:
Done.